terraform/cos/README.md

```diff
@@ -318,3 +318,40 @@ Then, use terraform to deploy the module:
 terraform init
 terraform apply
 ```
+
+### Telemetry correlation
```

Comment:
Telemetry correlation is an upstream feature, and is quite foundational to o11y, that I don't think we even need to mention it in a dedicated section here at all.